core/controllers/suggestion_test.py

```diff
@@ -198,47 +198,6 @@ def test_edit_state_content_suggestion_is_not_allowed(self):
             }, csrf_token=csrf_token, expected_status_int=400)
         self.logout()
 
-    def test_suggestion_to_exploration_handler_with_invalid_suggestion_id(self):
```

Comment:
A lot of these tests we shouldn't just delete since they test behavior not just related to SuggestionListHandler. For those tests, we should replace all SuggestionListHandler calls with equivalent query calls to replicate the same functionality, e.g. by calling `suggestion_services.query_suggestions`.